data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2022 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+mission "Remnant: Visit to Paradise 1"
+	name "Hoverboarding with Taely"
+	description "Take Taely to visit Vail, then return to Caelian."
+	minor
+	source
+		government "Remnant"
+	destination "Caelian"
+	stopover "Vail"
+	to offer
+		has "event: remnant: cognizance calm"
+		or
+			and
+				random > 30
+				not "Remnant: Visit to Paradise 1: defer"
+			and
+				random > 60
+				has "Remnant: Visit to Paradise 1: defer"
```

Comment:
Keep in mind that "Mission: defer" conditions aren't generated automatically.
```suggestion
			and
				random < 70
				not "Remnant: Visit to Paradise 1: defer"
			and
				random < 40
				has "Remnant: Visit to Paradise 1: defer"
```

Reply:
They're not? We have to manually create those when we use a defer?

Reply:
Yes.

Reply:
Alright, thanks for pointing that out. I'll add in an "on defer" clause to add in that statement.